micromanager_gui/main_window.py

```diff
@@ -258,6 +271,21 @@ def bin_changed(self):
             cd = self._mmc.getCameraDevice()
             self._mmc.setProperty(cd, "Binning", bins)
 
+    def _get_channel_group(self) -> Union[str, None]:
+        """
+        Get channelGroup falling back to Channel if not set, also
+        check that this is an availableConfigGroup.
+        """
+        chan_group = self._mmc.getChannelGroup()
```

Comment:
This is a low cost call - probably the same as accessing an attribute on self https://github.com/micro-manager/mmCoreAndDevices/blob/97f82b9ef02788d8b6e9aa70283eb10c7c645a6e/MMCore/MMCore.cpp#L3470-L3474